lib/integrations/slack.rb

```diff
@@ -46,11 +46,11 @@ def message_color
     private
 
     def message_text
+      description = run[:description].nil? ? '' : "(#{run[:description]}) "
       {
-
-        'run_completion' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> #{run[:status]}.",
-        'run_error' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> errored: #{run[:error_reason]}.",
-        'run_webhook_timeout' => "Your Rainforest run <#{payload[:frontend_url]}|##{payload[:id]}> timed out due to your webhook failing. If you need a hand debugging it, please let us know via email at team@rainforestqa.com.",
+        'run_completion' => "Your Rainforest Run <#{payload[:frontend_url]}|##{payload[:id]}> #{description}#{run[:status]}.",
```

Comment:
Is that making it clearer that way https://github.com/rainforestapp/rainforest-integrations/commit/f6375f07d9ca97d594743f63928eb7aa9fc796a6 ?